agent/testflinger_agent/client.py

```diff
@@ -105,6 +106,27 @@ def check_jobs(self):
             # Wait a little extra before trying again
             time.sleep(60)
 
+    def get_attachments(self, job_id: str, path: Path):
+        """Download the attachment archive associated with a job
+
+        :param job_id:
+            Id for the job
+        :param path:
+            Where to save the attachment archive.
+        """
```

Comment:
Return type and return value docstring could be added here.

Reply:
Attempting to address this, I realised that function doesn't need to return anything -- that was left over from a previous iteration. So I removed the return statement rather than modifying anything here.